nodewatcher/modules/monitor/http/survey/management/commands/export_survey_data.py

```diff
@@ -0,0 +1,137 @@
+import io
+import json
+import datetime
+
+from django.core.management import base
+
+from django_datastream import datastream
+
+
+class Command(base.BaseCommand):
+    help = "Consolidates the latest survey data within the specified time interval \
+           from all nodes and exports the meta graph as a JSON file."
+
+    def add_arguments(self, parser):
+
+        # Optional arguments
+        parser.add_argument(
+            '--datetime',
+            type=str,
+            action='store',
+            dest='datetime',
+            help="All the data will be collected between this UTC datetime \
+                 (format 'yyyy-mm-ddThh:mm:ss' (i.e. '2007-03-04T12:00:00 2007-04-10T12:00:00'), \
+                  and two hours preceding it."
+        )
+
+        parser.add_argument(
+            '-f',
+            '--filename',
+            type=str,
+            action='store',
+            dest='filename',
+            default='survey_export',
+            help="Optional filename",
+        )
+
+        parser.add_argument(
+            '--include-datetime-in-filename',
+            type=bool,
+            action='store',
+            dest='store_datetime',
+            default=False,
+            help="Include datetime in filename?",
+        )
+
+    def handle(self, *args, **options):
+        """
+        Exports the latest survey data graph as a JSON file into the root directory.
+        """
+
+        parsed_datetime = None
+        input_datetime = options['datetime']
```

Comment:
Keep variable names the same as key names. Do not rename things like that because it is harder to read the code. Name this option better from the beginning.